bitwarden_license/src/Commercial.Core/AdminConsole/Services/ProviderService.cs

```diff
@@ -647,6 +647,38 @@ public async Task DeleteAsync(Provider provider)
         await _applicationCacheService.DeleteProviderAbilityAsync(provider.Id);
     }
 
+    public async Task EnableAsync(Guid providerId)
+    {
+        var provider = await _providerRepository.GetByIdAsync(providerId);
+
+        if (provider is not { Enabled: false, Gateway: not null })
+        {
+            return;
+        }
+
+        provider.Enabled = true;
+        provider.RevisionDate = DateTime.UtcNow;
+
+        await _providerRepository.ReplaceAsync(provider);
+        await _applicationCacheService.UpsertProviderAbilityAsync(provider);
+    }
+
+    public async Task DisableAsync(Guid providerId)
+    {
+        var provider = await _providerRepository.GetByIdAsync(providerId);
+
+        if (provider is not { Enabled: true })
+        {
+            return;
+        }
+
+        provider.Enabled = true;
```

Comment:
This should be `false`

Reply:
Thanks for catching this! Just pushed an update